sdk/messaging/azeventgrid/response_types.go

```diff
@@ -16,8 +16,6 @@ type AcknowledgeCloudEventsResponse struct {
 
 // PublishCloudEventsResponse contains the response from method Client.PublishCloudEvents.
 type PublishCloudEventsResponse struct {
-	// Anything
-	Interface any
```

Comment:
I'm not opposed to this but I think the doc comment will need to be updated to omit the part about "In case of success, the server responds with an HTTP
// 200 status code with an empty JSON object in response"

Reply:
Fixed by just removing the 'schema' attribute, as you suggested. MUCH cleaner.